test-case-reporting/test/e2e.test.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Knex from 'knex';
+import nock from 'nock';
```

Comment:
We don't actually mock any network requests, so you can remove this import along with all `nock` message calls, and probably the `package.json` dependency too.

`nock` would be used if our app needed to, for example, make a POST request to the Travis API. We would "mock" that network request, and could expect certain properties about outgoing requests _from_ our app.

Reply:
Done.